src/color.scss

```diff
@@ -0,0 +1,103 @@
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
+
+/* stylelint-disable no-descending-specificity */
+
+// Handle highlighting the border during focus and invalid state
+@mixin border-highlight {
+  &:before {
+    border-bottom-color: transparent;
+  }
+}
+
+.wrapper {
+  border-color: transparent;
```

Comment:
So the idea here is to move all the color CSS into it's one stylesheet? Won't that make it difficult to maintain? 

What is the value of moving this to its own stylesheet versus having these styles in line with the others?